components/select/SelectBase.cs

```diff
@@ -810,6 +810,7 @@ protected async Task InvokeValuesChanged(SelectOptionItem<TItemValue, TItem> new
 
             if (ValuesChanged.HasDelegate)
             {
+                _selectedValues = newSelectedValues;
```

Comment:
It's better to put the assigment out of the 'if' block.

Reply:
sure, updated;